source/manual/business-readiness-publish-changes.html.md

```diff
@@ -33,12 +33,13 @@ The content item for the finder is updated from a [YAML file](govuk-app-deployme
 The content item for the business readiness finder is published by rummager. However the actual config is read in from [govuk-app-deployment-secrets](govuk-app-deployment-secrets).
 
 1. Update [govuk-app-deployment-secrets](govuk-app-deployment-secrets)
-2. Merge and deploy the change to [govuk-app-deployment-secrets](govuk-app-deployment-secrets)
-3. Run the `publishing_api:publish_document_finder` [rake task][staging-rake-task] in rummager to publish the changes:
+2. Merge the changes to [govuk-app-deployment-secrets](govuk-app-deployment-secrets)
+3. Re-deploy Rummager so it can pick up the changes in [govuk-app-deployment-secrets](govuk-app-deployment-secrets)
```

Comment:
I think it would be good to mention that you also need to redeploy finder-frontend if there are changes to the QA tool and email-alert-api if there are changes to the facets.

Reply:
Updated!